logs_http.module

```diff
@@ -15,20 +15,20 @@ use Drupal\logs_http\Logger\LogsHttpLogger;
  * always fatal: the execution of the script will stop as soon as the exception
  * handler exits.
  *
- * @param $exception
+ * @param \Throwable $exception
  *   The exception object that was thrown.
  *
  * @see _drupal_exception_handler()
  */
-function _logs_http_exception_handler($exception) {
+function _logs_http_exception_handler(\Throwable $exception) {
```

Comment:
Checking if tests will like the type hint. Also, we should probably move this to a helper class

Reply:
See #36 - this code is not needed anymore :)